scripts/install/linux/install-linux.sh

```
@@ -84,114 +84,16 @@ command_exists() {
-    
-    # Return the successfully parsed result
-    echo "$result"
-}
+COMMON_VALIDATION_LIB="$(dirname "$0")/../common/validation.sh"
+if [ ! -f "$COMMON_VALIDATION_LIB" ]; then
+    error "Common validation library not found: $COMMON_VALIDATION_LIB"
+    error "Please ensure the installation directory structure is intact."
+    exit 1
+fi
+# shellcheck source=../common/validation.sh
+source "$COMMON_VALIDATION_LIB"
```

Shellcheck (0.11.0)

[info] Not following: ../common/validation.sh was not specified as input (see shellcheck -x).

(SC1091)